### PR TITLE
Add 404 page

### LINKED ISSUE #3: Access denied instead of 404
when I navigate to a page like https://flakey-tests.ray.io/hello, I see:
> This XML file does not appear to have any style information associated with it. The document tree is shown below.
> 
> ---
> 
> ```xml
> <Error>
> <Code>AccessDenied</Code>
> <Message>Access Denied</Message>
> <RequestId>4D38D8DDD397DF71</RequestId>
> <HostId>hLdzL3qBQ2ln8O6pusXgovRCU7Ppa3pu1SLR4ZTuBrulRSqe8FLib6bqKgmYChqdSMiGbqPLUfo=</HostId>
> </Error>
> ```

I would expect a simple 404

### PR DESCRIPTION
**Pull request by sumanthratna**
Closes #3.

It's simple but gets the job done:
Screen Shot 2020-10-03 at 21 35 43

### CODE REVIEW

**Comment by simon-mo** [comment]
Actually this won't fix it just yet because the error message comes from cloudfront & s3. We will probably toggle something there.

**Comment by simon-mo** [comment]
image

Can we include the CSS or configure the font in this page? Somehow I'm getting serif font as default. (I was viewing the 404.html after `yarn build`, not the dev bundle)

File: js/src/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const NotFoundPage: React.FC = () => (
+  <h1
+    style={{
```
Comment (simon-mo) [inline js/src/pages/404.tsx:5]: fontFamily: sans-serif should be sufficient here.

**Comment by sumanthratna** [comment]
> I'm getting serif font as default. (I was viewing the 404.html after `yarn build`, not the dev bundle)

Yup, that happened to me too. Thanks for checking! How's this?
Screen Shot 2020-10-05 at 14 58 47
Screen Shot 2020-10-05 at 15 00 20

**Review by simon-mo — APPROVED** [review]
LGTM

**Pull request merged**